UPGRADING.md

```diff
@@ -523,7 +523,7 @@ For PUT, POST, PATCH, and DELETE requests where a non-empty body and a "Content-
 
 Grape no longer uses `multi_json` or `multi_xml` by default and uses `JSON` and `ActiveSupport::XmlMini` instead. This has no visible impact on JSON processing, but the default behavior of the XML parser has changed. For example, an XML POST containing `<user>Bobby T.</user>` was parsed as `Bobby T.` with `multi_xml`, and as now parsed as `{"__content__"=>"Bobby T."}` with `XmlMini`.
 
-If you were using `MultiJson.load`, `MultiJson.dump` or `MultiXml.parse`, you can substitute those with `Grape::Json.load`, `Grape::Json.dump`, `::Grape::Xml.parse`, or directly with `JSON.load`, `JSON.dump`, `XmlMini.parse`, etc.
+If you were using `MultiJson.load`, `MultiJson.dump` or `MultiXml.parse`, you can substitute those with `Grape::Util::Json.load`, `Grape::Util::Json.dump`, `::Grape::Util::Xml.parse`, or directly with `JSON.load`, `JSON.dump`, `XmlMini.parse`, etc.
```

Comment:
Are the namespace changes of `Grape::Json` to `Grape::Util::Json` breaking or does `Grape::Json` still work? 

Either way add a new section in UPGRADING that says to use `Grape::Util::Json` instead of `Grape::Json`.

Reply:
I can keep the old namespace. I'll just need to move files from the `utils` folder and remove the `utils` modules.